lib/runcible/base.rb

```diff
@@ -41,6 +41,9 @@ def call(method, path, options = {})
       client_options[:timeout] =  clone_config[:timeout] if clone_config[:timeout]
       client_options[:open_timeout] =  clone_config[:open_timeout] if clone_config[:open_timeout]
       client_options[:verify_ssl] =  clone_config[:verify_ssl] unless clone_config[:verify_ssl].nil?
+      client_options[:ssl_client_cert] => clone_config[:ssl_client_cert] unless clone_config[:ssl_client_cert].nil?
+      client_options[:ssl_client_key] => clone_config[:ssl_client_key] unless clone_config[:ssl_client_key].nil?
+      client_options[:ssl_ca_file] => clone_config[:ssl_ca_file] unless clone_config[:ssl_ca_file].nil?
```

Comment:
Also worth thinking about is if we should throw some sort of configuration error if only a subset of these are set. Most likely, requests will just outright fail so we may want to prevent users from having to debug that when they simply misconfigured Runcible.